app/jp_mobile_carriers.py

```diff
@@ -147,24 +147,31 @@ def look_up_number(number):
         Exception when failed to scrape.
     """
     # Scrape Docomo's gateway page and get a hidden time stamp param.
-    scrape = urllib2.urlopen(DOCOMO_URL).read()
-    hidden_param = DOCOMO_HIDDEN_RE.findall(scrape)[0]
-
-    # Encode the number and the above param as POST data
-    data = get_docomo_post_data(number, hidden_param)
-    encoded_data = urllib.urlencode(data)
-    # Scrape Docomo's answer on the number
-    scrape = urllib2.urlopen(DOCOMO_URL, encoded_data).read()
-
-    # Extract a further redirect url, if any.
-    url = extract_redirect_url(scrape)
-    if url:
-        return url
-    elif docomo_has_messages(scrape):
-        # Checks if Docomo has messages for the number, and returns the url
-        # for Docomo if it does.
-        return DOCOMO_URL + '?' + encoded_data
-
+    try:
+        scrape = urllib2.urlopen(DOCOMO_URL).read()
+    except (urllib2.URLError, httplib.HTTPException, socket.error) as e:
+        raise e
```

Comment:
These "except" clauses just throw the original exception, so I believe it's equivalent to the original code. Do you have any reason to add these clauses?